network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -89,131 +185,203 @@ class ProductPage(FoundationMetadataPageMixin, Page):
     signup_requires_email = ExtendedYesNoField(
         help_text='Does this product requires providing an email address in order to sign up?'
     )
-
     signup_requires_phone = ExtendedYesNoField(
         help_text='Does this product requires providing a phone number in order to sign up?'
     )
-
     signup_requires_third_party_account = ExtendedYesNoField(
         help_text='Does this product require a third party account in order to sign up?'
     )
-
     signup_requirement_explanation = models.TextField(
         max_length=5000,
         blank=True,
         help_text='Describe the particulars around sign-up requirements here.'
     )
 
     # How does it use this data?
-
     how_does_it_use_data_collected = models.TextField(
         max_length=5000,
         blank=True,
         help_text='How does this product use the data collected?'
     )
-
     data_collection_policy_is_bad = models.BooleanField(
         default=False,
         verbose_name='Privacy ding'
     )
 
     # Privacy policy
-
     user_friendly_privacy_policy = ExtendedYesNoField(
         help_text='Does this product have a user-friendly privacy policy?'
     )
 
     # Minimum security standards
-
     show_ding_for_minimum_security_standards = models.BooleanField(
         default=False,
         verbose_name="Privacy ding"
     )
-
     meets_minimum_security_standards = models.BooleanField(
         null=True,
         help_text='Does this product meet our minimum security standards?',
     )
-
     uses_encryption = ExtendedYesNoField(
         help_text='Does the product use encryption?',
     )
-
     uses_encryption_helptext = models.TextField(
         max_length=5000,
         blank=True
     )
-
     security_updates = ExtendedYesNoField(
         help_text='Security updates?',
     )
-
     security_updates_helptext = models.TextField(
         max_length=5000,
         blank=True
     )
-
     strong_password = ExtendedYesNoField()
-
     strong_password_helptext = models.TextField(
         max_length=5000,
         blank=True
     )
-
     manage_vulnerabilities = ExtendedYesNoField(
         help_text='Manages security vulnerabilities?',
     )
-
     manage_vulnerabilities_helptext = models.TextField(
         max_length=5000,
         blank=True
     )
-
     privacy_policy = ExtendedYesNoField(
         help_text='Does this product have a privacy policy?'
     )
-
     privacy_policy_helptext = models.TextField(  # REPURPOSED: WILL REQUIRE A 'clear' MIGRATION
         max_length=5000,
         blank=True
     )
 
-    """
-    privacy_policy_links = one to many, defined in PrivacyPolicyLink
-    """
```

Comment:
still a good idea to keep this comment, because reverse relational property assignment is black goddamn Django magic and anyone new to django/wagtail will be confused AF about why this property seems to exist, when it's not even defined for this class.

Reply:
Good call, this is now an Orderable though, so I'll update the comment accordingly.